.circleci/config.yml

```diff
@@ -28,24 +28,38 @@ jobs:
   build:
     machine:
       image: ubuntu-1604:201903-01 
-    working_directory: ~/inbox
+    working_directory: ~/sync-engine
 
     steps:
       - checkout
 
       - run:
-          name: build nylas image
-          command: docker-compose build nylas
+          name: build sync-engine image
+          command: docker-compose build app
 
       - run:
           name: run tests
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS
-            docker-compose run nylas bash -ec '
+            docker-compose run app bash -ec '
               pip install -e . \
               && bin/wait-for-it.sh mysql:3306 \
               && NYLAS_ENV=test py.test tests/
             '
+      - run:
```

Comment:
Copied this from https://github.com/closeio/authalligator/blob/master/.circleci/config.yml#L98-L113